app/views/exams/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= simple_form_for(@exam) do |f| %>
+  <%= f.error_notification %>
+
+  <div class="form-inputs">
+    <%= f.input :score %>
+    <%= f.input :exam_date %>
```

Comment:
I would copy the datepickers used in the tutor form. Roberta wants typeable dates.